features/ouders.feature

```diff
@@ -19,236 +19,234 @@ Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie
-    | ouderAanduiding                         | 1        |
-    En heeft de ouder met ouderAanduiding '1' de volgende gegevens
-    | naam                                    | waarde   |
-    | ouderAanduiding                         | 1        |
-    | datumIngangFamilierechtelijkeBetrekking | 00000000 |
-    En de ouder met ouderAanduiding '1' heeft GEEN 'geslacht' gegevens
-    En de ouder met ouderAanduiding '1' heeft GEEN 'geboorte' gegevens
-    En heeft de ouder met ouderAanduiding '1' alleen de volgende 'naam' gegevens
-    | naam          | waarde |
-    | geslachtsnaam | .      |
+    Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
+    | naam                                                | waarde        |
+    | ouderAanduiding                                     | 1             |
+    | datumIngangFamilierechtelijkeBetrekking.type        | DatumOnbekend |
+    | datumIngangFamilierechtelijkeBetrekking.onbekend    | true          |
+    | datumIngangFamilierechtelijkeBetrekking.langFormaat | onbekend      |
 
   @gba
   Scenario: met fields vragen om gegevens zonder waarde
-    Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende ouders in de registratie
-    | Categorie | Voornamen (02.10)  | Geslachtsnaam | Geboortedatum (03.10) | Geslachtsaanduiding (04.10) | Datum ingang familierechtelijke betrekking (62.10) |
-    | 2         |                    | Jansen        |                       | V                           | 20190614                                           |
-    | 3         |                    |               |                       |                             |                                                    |
-    En in categorie 3 komt alleen groep 81 en groep 85 voor
+    Gegeven de persoon met burgerservicenummer '000007898' heeft een ouder '1' met de volgende gegevens
+    | geslachtsnaam (02.40) | geslachtsaanduiding (04.10) | datum ingang familierechtelijke betrekking (62.10) |
+    | Jansen                | V                           | 20190614                                           |
+    En de persoon heeft een ouder '2' met de volgende gegevens
+	| aktenummer (81.20) | ingangsdatum geldigheid (85.10) |
+	| 2•E0001            | 20160518                        |
     Als personen wordt gezocht met de volgende parameters
     | naam                | waarde                                 |
     | type                | RaadpleegMetBurgerservicenummer        |
-    | burgerservicenummer | 555550001                              |
-    | fields              | ouders.voornamen,ouders.geboorte.datum |
-    Dan heeft de persoon met burgerservicenummer '555550001' exact 1 ouders
-    En heeft de ouder GEEN gegevens
+    | burgerservicenummer | 000007898                              |
+    | fields              | ouders.naam.voornamen,ouders.geboorte.datum |
+    Dan heeft de response een persoon met een 'ouder' zonder gegevens
```

Comment:
met een leeg 'naam' object

Reply:
Het scenario `Gevraagd gegevensgroep veld heeft geen waarde voor de persoon` in fields.feature geeft je gelijk. Waar kan ik de Rule vinden die dit gedrag beschrijft? Ik vind hem niet in dat feature bestand.